### ISSUE TITLE
Synse is doing writes very rapidly, overwhelming devices.

### ISSUE THREAD

**Issue by MatthewHink** [vapor-ware/synse-sdk#489]
This is happening on the iPIO-8s. This is causing total communication loss with the device.

The plugin configuration is here, emphasis on serial:
```
[mhink@ion ~/src/vapor/helmfiles (mhink-dxp-changes)]$ kubectl describe cm north-side-dxp-config | tail -n 20

Data
====
config.yml:
----
debug: true
network:
  address: :5009
  type: tcp
settings:
  cache:
    enabled: true
    ttl: 5m
  **mode: serial**
  read:
    interval: 5s
  write:
    interval: 1s
version: 3
Events:  <none>
```

Logs:
[synse-parallel-writes.txt](https://github.com/vapor-ware/synse-sdk/files/6300157/synse-parallel-writes.txt)

There is a delay here, but I do not see where it is used other than for the log statement.
https://github.com/vapor-ware/synse-sdk/blob/f4b2910b9a8ba8d02e8d464d194137e875cce4fc/sdk/scheduler.go#L647-L651

**Comment by edaniszewski** [comment]
I think I see whats happening here.

The behavior of the config options are suuuuper obvious just based on their names alone, so `interval` isn't the interval between reads, but the interval at which the write loop runs. (in serial mode the SDK alternates between running the read loop and write loop). It does not control the frequency at which items are written to within that loop. For that we should set the [`delay`](https://github.com/vapor-ware/synse-sdk/blob/master/sdk/config/plugin.go#L255) config option, which adds a sleep between writes in a single loop run

**Comment by edaniszewski** [comment]
but.. now that I'm digging through the source, you're right.. I'm not seeing where `delay` is set for writes. I see if for reads, just not in the write block portion

I'm going to dig through some git history to see if it has ever existed or not.. this feels like a bug.

**Comment by GuessWhoSamFoo** [comment]
Closed via #490
